src/services/CardsService.ts

```diff
@@ -48,4 +48,20 @@ export class CardsService {
 			return response;
 		});
 	}
+
+	public updateOne(id: number, update: object) {
```

Comment:
This needs a call signature, 

```
public updateOne(id: number, update: object): <call signature goes here> {

}
```

Was my bad for not defining one above, but we should make an interface for the response and then set this signature to that, as well as the `delete` and `create` methods above.